src/pages/contact/index.page.tsx

```diff
@@ -1,14 +1,15 @@
 import React from 'react';
 import Head from 'next/head';
+import ContactUs from '../../components/ContactUs';
 
 export default function ContactPage() {
   return (
     <>
       <Head>
         <title>Reactjs Developer Community in Kenya - Contact</title>
       </Head>
-      <main className="flex justify-center items-center min-h-screen">
-        <h1 className="text-white">Contact page will be here</h1>
+      <main className="min-h-screen bg-white">
+        <ContactUs />
```

Comment:
We should include this component in the `src/pages/index.page.tsx` page as it appears in the design. The ContactPage will be deleted in a future PR.

Reply:
I see. Fixed via [482856a](https://github.com/reactdeveloperske/reactdevske-website/pull/102/commits/482856ad7a936a7bbb39002ae1aa89ef9228e442)